### PR TITLE
Init readme

### PR DESCRIPTION
**Pull request by meile18**
## This introduces a breaking change

- [x] No

## This Pull Request implements

init readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+## Important aliases
+license name:
+- `BSD` without clause = 4 clause BSD ([source](https://en.wikipedia.org/wiki/BSD_licenses#Terms))
+
+classifier:
+- None for now
+
+## Run solver locally
+
+Often, it is useful to run adviser locally to experiment or verify your changes in implementation. You can do so easily
```
Comment (fridex) [inline README.md:25]:
```suggestion
Often, it is useful to run license-solver locally to experiment or verify your changes in implementation. You can do so easily
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+## How it works
+
+license-solver detect licenses and classifiers from input files/directory. If the solver can't find a license or
+classifier related to the supported name, then the result is for them `UNKNOWN`, for undetected license version
+is `UNDETECTED`. Some licenses do not have versions, and their naming is `LICENSE-WITHOUT-VERSION`. All dictionaries with the most used aliases are stored in thoth/license_solver/data. <br>
+After successful detection, license and classifier will be compared. If they don't match, then in final JSON will be
+warning: True. In case if it is missing license or classifier, comparing will be aborted. <br>
+In final will be printed JSON on STDOUT.
+
+## Important aliases
```
Comment (fridex) [inline README.md:16]: It is not directly clear to me what aliases are in this section.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
-3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
-4. We have a public service configuration now accessible for everyone.
-5. If you are writing a Python module, [Kebechet](https://github.com/marketplace/khebhut) could manage the releases of your packages for you.
-6. If credentials are provided, Python module releases could be published to [`PyPI`](https://pypi.org/) by [Kebechet](https://github.com/marketplace/khebhut).
+This repository serves for license detection and classifier detection from PyPI metadata.
 
-Dependencies should be managed using `pipenv` (`Pipfile`, and the `Pipfile.lock` could be created by `thamos advise`), `pip3` and a `requirements.txt` files could be used.
+## How it works
+
+license-solver detect licenses and classifiers from input files/directory. If the solver can't find a license or
```
Comment (fridex) [inline README.md:9]:
```suggestion
license-solver detects licenses and classifiers from input files/directory. If the solver can't find a license or
```

**Review by fridex — COMMENTED** [review]
nice! 💯 few minor things 👍🏻

File: README.md (hunk 2 above)
Comment (meile18) [reply README.md:16]:
```suggestion
## Important aliases
This section provides information about specific implementation detail, which can be not obviously clear.
- `BSD` license name in metadata without specific clause is `4-clause BSD` in license-solver implementation ([source](https://en.wikipedia.org/wiki/BSD_licenses#Terms))
```
it is better?

Comment (fridex) [reply README.md:16]: Sounds better 👍🏻

**Comment by meile18** [comment]
@fridex, do you think I should add another section or some changes?

**Review by fridex — APPROVED** [review]
lgtm, thanks! 👍🏻 💯

**Pull request merged**